src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java

```diff
@@ -206,6 +238,12 @@ public AppComputeSecretController(AuthorizationService authorizationService,
                                                                             @PathVariable String requesterAddress,
                                                                             @PathVariable String secretKey,
                                                                             @RequestBody String secretValue) {
+        if (!secretKeyPattern.matcher(secretKey).matches()) {
```

Comment:
Same here, we can bundle check inside a method to be reused in both parent methods

Reply:
done in d0033ed